packages/auth-server/package.json

```diff
@@ -18,5 +18,8 @@
     },
     "devDependencies": {
         "@types/cors": "^2.8.17"
+    },
+    "peerDependencies": {
+        "tslib": "^2.6.2"
```

Comment:
this one is not published to npm so it imho doesn't need this line? 
on the other hand, all packages that are published to npm shlould have tslib listed as peer dependency, am I right?

Reply:
Yea, it should be listed for all published but I think it won't hurt here either.